### ISSUE TITLE
Missing transform functions (rotateX, rotateY, perspective)

### ISSUE THREAD

**Issue by jeremiahmmv2** [jeremyckahn/rekapi#37]
I'm planning to recreate animate.css animations with Rekapi and noticed that the rotateX, rotateY and perspective functions are missing from the transformFunctions array. 

I was attempting to recreate the flipInX animation when I noticed it.

```css
@keyframes flipInX {
    0% {
        transform: perspective(400px) rotateX(90deg);
        opacity: 0;
    }

    40% {
        transform: perspective(400px) rotateX(-10deg);
    }

    70% {
        transform: perspective(400px) rotateX(10deg);
    }

    100% {
        transform: perspective(400px) rotateX(0deg);
        opacity: 1;
    };
}
```

I added them to transformFunctions in my local version and it seems to work fine in Chrome. Is there a reason they're not available by default?

**Comment by jeremyckahn** [comment]
Hi @jeremiahmmv2, thanks for reporting!  Nope, I must have just forgotten those functions.  It sounds like you have it working — could you please make a Pull Request with your changes?  I'm not sure what default order is preferable for those new functions to be in (not `perspective`, anyways), and it would be best to use your `transformFunctions` list since you have an actual use case you are working with.

**Comment by jeremiahmmv2** [comment]
If I get a chance before you, I'll make a pull request. This order is working for me:

```javascript
  var transformFunctions = [
    'translateX',
    'translateY',
    'translateZ',
    'scale',
    'scaleX',
    'scaleY',
    'perspective',
    'rotate',
    'rotateX',
    'rotateY',
    'skewX',
    'skewY'];
```

**Comment by jeremyckahn** [comment]
The fix is available in [1.5.1](https://github.com/jeremyckahn/rekapi/releases/tag/1.5.1).  Thanks again!

If your Rekapi port of animate.css is going to be open source, please let me know when it is available so I can link to it in the Rekapi documentation.
